bloom-housing/bloom
# 3030/what to expect visual updates

# Pull Request Template

## Issue Overview

This PR addresses #3030


## Description

- `open waitlist`, `vacant units` headers will now display as Tag pills
  - for some reason Tag pill `secondary` color is `primary` (i didn't change that)
- i changed `Available Unit` to `Vacant Unit` in translations, so it means it will also change in cases not mentioned in task like partners page listing table. For me that unification seem reasonable, but not sure if i should create new translation for that 🤔 **-> that one make test fail**
- also now i see that there is `vacant units available` in designs i just changed to `vacant units`. Tbh totally don't know how to handle adding new field in translation. 🤷 Would need some help 😅 
- again now i see we have that weird indent:
<img width="324" alt="Screenshot 2022-09-14 at 17 11 25" src="https://user-images.githubusercontent.com/9580543/190193848-4dc3ff43-30fc-4d1a-8407-ecdb9211269f.png">
should i change that so if there is one element then just don't make that indent or sth (it uses reusable component if i get it right)?

## How Can This Be Tested/Reviewed?

http://localhost:3000/listings -> to check pill tags (check both `open waitlist`, `vacant units`),
go to listing with `vacant units` text changes

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
